src/Calculator.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+
+const result = 0;
+
+const Display = () => (
```

Comment:
- [ ] It would be better if we create a different file for the display component, it is not best practices to have several components in one file. Also keep in mind that the `calculator.js` component should have a child component
![Screenshot from 2023-08-16 08-03-50](https://github.com/Nessrine88/MathMagicians/assets/46388113/31831d27-91a8-47fc-a632-f1afeb0fdcca)